### DISCUSSION TITLE
La mort

### DISCUSSION THREAD

**Discussion by aureochocob0n** [Crownicles/Crownicles#2573] [Nouvelles idées]
Je propose que lorsque nous mourrons au-delà du lv100, nous perdions de l'xp en plus des points, assez pour nous enlever 1 ou plusieurs niveau.

Pourquoi après le lv100? 
Parce qu'à ce niveau, on connaît bien le jeu et contrairement aux bas lv, on a un nombre de pv assez important. Faire perdre de l'xp aux plus bas lv pourrait les dégoûter du jeu

**Comment by Cirpacha** [comment]
Encore une idée pour dégoûter les hauts level, yes
Et la reconnaissance dans tout ça ? ceux qui s'investissent autant dans le bot ne méritent pas de s'en prendre autant dans la face^^
On a déjà des gobelets avec des longues altérations d'état, une île quasi impossible à finir, un progrès qui se voit à peine, etc., n'en rajoutons pas.
Surtout si c'est pour enlever "un ou plusieurs niveaux" à chaque fois, tu ne sembles pas réaliser le temps de jeu nécessaire à l'obtention de ce dit niveau. L'expérience est la seule ressource que les joueurs ne peuvent pas perdre, et c'est pour une raison. Il faut garder assez de caractéristiques pour différencier les points et l'xp.

**Comment by romain22222** [comment]
j'ai encore mieux : au niveau 100+, on débloque la véritable mort : si on meurt on est reset

**Reply by BastLast** [reply]
le gameplay de voisin be like

**Comment by StrangerIM** [comment]
Bonjour non
